src/ansys/dpf/core/incremental.py

```diff
@@ -0,0 +1,237 @@
+"""
+.. _ref_incremental:
+
+Incremental
+========
+"""
+
+from ansys.dpf import core
+
+from typing import Dict, Any
+
+
+class IncrementalHelper:
+    def __init__(
+        self,
+        start_op: core.Operator,
+        end_op: core.Operator,
+        scoping: core.Scoping,
+        scoping_pin: int = None,
+    ):
+        """
+        Given the first and the last operator of a workflow, as well as the scoping.
+
+        This class can be used to simplify the use of incremental operators, and automatically
+        enable to incrementally evaluate a workflow.
+
+        Under the constraint that the end_op supports incremental evaluation.
+        """
+        # input operator should accept a scoping
+        # last operator should support incremental evaluation
+        #     but it should be permissive too (bad doc/spec whatever)
```

Comment:
(for bad docs or specs?) Not sure what the content in this bracket means.

Reply:
Yes, it was unclear.

Basically this function is used for validation, but in the case this function doesn't validate, it should not create an error but simply inform the user through a warning.
Because the operator that it is trying to validate may have a **bad specification or documentation or whatever the inconsistency is**.

Please tell me if the current comments are clearer in that regard.